package.json

```diff
@@ -7,31 +7,42 @@
     "eslint",
     "eslintconfig"
   ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/simplabs/eslint-config-simplabs.git"
+  },
+  "license": "MIT",
+  "author": "simplabs GmbH <info@simplabs.com>",
   "homepage": "https://github.com/simplabs/eslint-config-simplabs#readme",
   "bugs": {
     "url": "https://github.com/simplabs/eslint-config-simplabs/issues"
   },
-  "license": "MIT",
-  "author": "simplabs GmbH <info@simplabs.com>",
   "main": "configs/ember.js",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/simplabs/eslint-config-simplabs.git"
-  },
   "scripts": {
     "test": "node .",
     "nodetest": "mocha node-tests --recursive"
   },
   "dependencies": {
+    "ember-cli-htmlbars": "^4.0.0",
```

Comment:
why? we don't have any templates in here, right?

Reply:
I ran `ember-cli-update` which added all the default dependencies but of course we don't need them in this case - removed 👍